sympl/_core/util.py

```diff
@@ -639,6 +638,141 @@ def __getattr__(self, item):
         return getattr(self._prognostic, item)
 
 
+class ScaledInputOutputWrapper(object):
```

Comment:
Rename to "ScalingWrapper", this is a bit verbose (and doesn't reflect that it can scale tendencies and diagnostics).